taskvine/src/manager/vine_manager.h

```diff
@@ -116,7 +116,7 @@ struct vine_manager {
 	struct hash_table *worker_blocklist; /* Maps hostname -> vine_blocklist_info */
 	struct hash_table *factory_table;    /* Maps factory_name -> vine_factory_info */
 	struct hash_table *workers_with_watched_file_updates;  /* Maps link -> vine_worker_info */
-	struct hash_table *workers_with_complete_tasks;  /* Maps link -> vine_worker_info */
+	struct list       *workers_with_complete_tasks;  /* List of worker->hashkey. Only hashkey in case vine_worker_info is freed. */
```

Comment:
Elaborate the comment a bit here -- had to read it a few times before I understood the sense of it.